.laminas-ci.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extensions": [
+        "curl",
+        "dom",
+        "gd",
+        "libxml"

```

Comment:
Afaik, `libxml` is bundled with PHP.
Same for `gd`, `dom` and `curl`.

Try to drop these to see if build passes. 👍🏼

Reply:
You are correct in that curl, dom and libxml come bundled but GD is not. I have updated the configuration to reflect this.